metrics/web/test/dashboard/presentation/widgets/no_search_results_placeholder_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/metrics_table/theme_data/project_metrics_table_theme_data.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/metrics_table/theme_data/project_metrics_tile_theme_data.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/metrics_theme_data.dart';
+import 'package:metrics/common/presentation/text_placeholder/theme/theme_data/text_placeholder_theme_data.dart';
+import 'package:metrics/dashboard/presentation/strings/dashboard_strings.dart';
+import 'package:metrics/dashboard/presentation/widgets/no_search_results_placeholder.dart';
+
+import '../../../test_utils/finder_util.dart';
+import '../../../test_utils/metrics_themed_testbed.dart';
+
+void main() {
+  const backgroundColor = Colors.red;
+  const textStyle = TextStyle(color: Colors.red);
+
+  const metricsThemeData = MetricsThemeData(
+    projectMetricsTableTheme: ProjectMetricsTableThemeData(
+      projectMetricsTileTheme: ProjectMetricsTileThemeData(
+        backgroundColor: backgroundColor,
+        textStyle: textStyle,
+      ),
+    ),
+    textPlaceholderTheme: TextPlaceholderThemeData(
+      textStyle: textStyle,
+    ),
+  );
+
+  group("NoSearchResultsPlaceholder", () {
+    testWidgets(
+      "displays the no search results placeholder",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const _NoSearchResultsPlaceholderTestbed());
+
+        expect(find.text(DashboardStrings.noSearchResults), findsOneWidget);
+      },
+    );
+
+    testWidgets(
+      "applies the background color from the metrics theme",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _NoSearchResultsPlaceholderTestbed(
+            metricsThemeData: metricsThemeData,
+          ),
+        );
+
+        final decoration = FinderUtil.findBoxDecoration(tester);
+
+        expect(decoration.color, equals(backgroundColor));
+      },
+    );
+
+    testWidgets(
+      "applies the text style from the metrics theme",
```

Comment:
It seems like we should not test it here because we have such a test in tests for the TextPlaceholder widget. Also, it will be great to tests that this widget applies the border color from the metrics theme.